test/senders/beanstlakd_sender_test.go

```diff
@@ -0,0 +1,106 @@
+package senders_test
+
+import (
+	"testing"
+
+	models "github.com/shoplineapp/captin/models"
+
+	"github.com/shoplineapp/captin/senders"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestBeanstalkdSender_SendEvent_Success_WithIPAndPort(t *testing.T) {
```

Comment:
https://dave.cheney.net/2019/05/07/prefer-table-driven-tests
consider to use table driver tests